modeling-cmds/src/units.rs

```diff
@@ -92,8 +94,13 @@ impl UnitLength {
 #[cfg_attr(feature = "python", pyo3::pyclass, pyo3_stub_gen::derive::gen_stub_pyclass_enum)]
 pub enum UnitAngle {
     /// Degrees <https://en.wikipedia.org/wiki/Degree_(angle)>
+    #[default]
+    #[serde(rename = "deg")]
+    #[display("deg")]
```

Comment:
just want to make sure that this wont break anything elsewhere but lgtm if not! you all have more context than i do here